src/components/footer/index.jsx

```diff
@@ -11,6 +12,24 @@ import "./index.scss";
 
 const Footer = () => {
   const { locale, t } = useTranslation();
+  useEffect(() => {
+    const script = document.createElement('script');
+
+    script.src = "https://widget.kapa.ai/kapa-widget.bundle.js";
+    script.async = true;
+    script.dataset.websiteId = "e3268e5d-c0f1-4e71-819c-c60ebb2215a7";
+    script.dataset.projectName = "Apache DolphinScheduler";
+    script.dataset.projectColor = "#0097E0";
+    script.dataset.projectLogo = "images/logo_400x400.jpg";
```

Comment:
Does the logo already exist under our project?

Reply:
Not yet. This PR will add this logo. Will it be deployed automatically after the PR merge?